docs/en/mavlink/protocols.md

```diff
@@ -0,0 +1,53 @@
+# MAVLink Microservices (Protocols)
+
+MAVLink "microservices" are a protocols that use multiple messages exchanged between components to communicate more complicated information.
+For example, the [Command Protocol](https://mavlink.io/en/services/command.html) provides an efficient mechanism for packaging a command in a (particular) message and receiving acknowledgement of the command in another message.
+
+MAVLink microservices are documented the [MAVLink Guide](https://mavlink.io/en/services/) (this is not exhaustive: not all messages are grouped into protocols and not all protocols are documented).
+
+This section lists the services known to be supported/not supported by PX4 in this version.
+
+## Supported Microservices
+
+These services are known to be supported in some form:
+
+- [Battery Protocol](https://mavlink.io/en/services/battery.html)
+  - [BATTERY_STATUS](https://mavlink.io/en/messages/common.html#BATTERY_STATUS) and [BATTERY_INFO](https://mavlink.io/en/messages/common.html#BATTERY_STATUS) are streamed.
+- Camera Protocols
+  - [Camera Protocol v2](https://mavlink.io/en/services/camera.html)
+    - [Camera Definition](https://mavlink.io/en/services/camera_def.html)
+  - [Camera Protocol v1 (Simple Trigger Protocol)](https://mavlink.io/en/services/camera_v1.html)
```

Comment:
```suggestion
```

Reply:
I guess we should remove it if it isn't needed!